test/09_platform_selector_problem/cibuildwheel_test.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+import os 
+import sys
+import subprocess 
+import utils 
+
+
+def test_wrong_identifier_py2_py34():
+    if not (sys.version_info[0] == 2 or (sys.version_info[0] == 3 and sys.version_info[1] == 4)):
+        pytest.skip("test for python 2.7 and 3.4")
+    project_dir = os.path.dirname(__file__)
+
+    with pytest.raises(subprocess.CalledProcessError) as excinfo:
+        utils.cibuildwheel_run(project_dir, add_env={'CIBW_BUILD':'py368-*'})
```

Comment:
If Python 2.7 now works, these tests can be simplified?

Reply:
I do not understand. There is no `subprocess.run` in python 2.7.

Reply:
Sorry, I got confused. I thought the `universal_newlines` fixed this, but indeed, `run` is still missing. My bad!